src/SFA.DAS.ApplyService.Web.UnitTests/Controllers/RoatpApplicationPreambleControllerTests.cs

```diff
@@ -274,12 +278,12 @@ public void Before_each_test()
             Mapper.AssertConfigurationIsValid();
         }
 
-        [TestCase("")]
-        [TestCase(null)]
-        [TestCase("1234567")]
-        [TestCase("123456789")]
-        [TestCase("1234567A")]
-        public void Validation_error_is_triggered_if_UKPRN_not_in_correct_format(string ukprn)
+        [TestCase("", _validationTypeMissing)]
+        [TestCase(null, _validationTypeMissing)]
+        [TestCase("1234567",_validationTypeInvalid)]
+        [TestCase("123456789", _validationTypeInvalid)]
+        [TestCase("1234567A", _validationTypeInvalid)]
+        public void Validation_error_is_triggered_if_UKPRN_not_in_correct_format(string ukprn, string validationMessageType)
```

Comment:
The test names are not according to recommendations as described [here](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#naming-your-tests)

Rename suggestion SearchByUkprn_InvalidInput_RedirectsToEnterUkrpnWithRespectiveValidationMessage